template-configuration.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Tags" : {
+    "custom:mytag" : "myvale"
```

Comment:
myvale? :P

Reply:
Jeg må innrømme at jeg vet ikke hva det er for noe. Tror Thomas har lagt det inn at templaten kan få en tag også eller det er en konfig for tagging da man release ting i en pipeline-step?
Bør jeg fjerne det siden vi ikke har bruk for det?